run-mockup.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
I don't think we want this shell script in here since it references another tool. Is this really needed?